MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift

```diff
@@ -71,33 +71,60 @@ private func getUnusedLogFiles() -> [URL] {
     return allLogFiles
 }
 
-func logCleanup(maxMB: UInt64) {
-    DispatchQueue.global().async {
-        let maxBytes = maxMB * 1000000
+private func logCleanup(maxMB: UInt64) {
+    let maxBytes = maxMB * 1000000
 
-        let fileManager = FileManager.default
+    let fileManager = FileManager.default
 
-        var loopIterationsFailsafe = 0
-        while totalBytes(files: getUnusedLogFiles()) > maxBytes && loopIterationsFailsafe < 10 {
-            loopIterationsFailsafe = loopIterationsFailsafe + 1
+    var loopIterationsFailsafe = 0
+    while totalBytes(files: getUnusedLogFiles()) > maxBytes && loopIterationsFailsafe < 10 {
+        loopIterationsFailsafe = loopIterationsFailsafe + 1
 
-            guard let oldestFile = getUnusedLogFiles().last else {
-                TariLogger.error("Failed to get oldest log file")
-                break
-            }
+        guard let oldestFile = getUnusedLogFiles().last else {
+            TariLogger.error("Failed to get oldest log file")
+            break
+        }
 
-            guard fileManager.fileExists(atPath: oldestFile.path) else {
-                TariLogger.error("Oldest log file does not exist")
-                break
-            }
+        guard fileManager.fileExists(atPath: oldestFile.path) else {
+            TariLogger.error("Oldest log file does not exist")
+            break
+        }
 
-            do {
-                try fileManager.removeItem(at: oldestFile)
-                TariLogger.info("Deleting log file: \(oldestFile.lastPathComponent)")
-            } catch {
-                TariLogger.error("Failed to delete log file", error: error)
-                break
+        do {
+            try fileManager.removeItem(at: oldestFile)
+            TariLogger.info("Deleting log file: \(oldestFile.lastPathComponent)")
+        } catch {
+            TariLogger.error("Failed to delete log file", error: error)
+            break
+        }
+    }
+}
+
+private func bugReportZipFilesCleanup() {
```

Comment:
I think you can safely delete any zip files at the beginning of a new log mail request (thinking a zip is prepped only for a single request) - this way you'd have max 1 zip file sitting at any time.

Reply:
A different story if you'd like to access them later of course.

Reply:
Originally that's what I was doing but then I decided if they never report another bug then it's a few mb that stays forever with no purpose

Reply:
I see you're right - now I have to do the same thing on Android 😄

Reply:
Awesome 👍